### DISCUSSION TITLE
<student pack>

### DISCUSSION THREAD

**Discussion by pooyash97** [community/community#39470] [GitHub Education]
### Student starter pack

Bug

so, in the discount section when it tries to recieve my location it takes a lot of time and it doesn't do anything afterwards.

**Comment by samus-aran** [comment]
Hi @pooyash97 👋 

There has been changes to the student application form which now prompts you to enable browser geolocation. This step is now mandatory before proceeding the next step in the application process. Please cancel your current application and re-try and let us know how you get on 👍
